buildenv/jenkins/variables/defaults.yml

```diff
@@ -550,7 +546,7 @@ x86-64_windows:
   node_labels:
     build: 'ci.role.build && hw.arch.x86 && sw.os.windows'
   build_env:
-    vars: 'PATH+TOOLS=/cygdrive/c/openjdk/LLVM64/bin:/cygdrive/c/openjdk/nasm-2.13.03'
+    vars: 'PATH+TOOLS=/cygdrive/c/openjdk/LLVM64/bin'
```

Comment:
Looking at a machine it contains nasm 2.13.01 but the minimum is 2.13.03
i.e. the machines need to be updated before this change is made. I expect @jdekonin will be reluctant to update them at this time without good reason, as there are other priorities.

Reply:
I don't know where the requirement for 2.13.03 came from; I was able to build all windows versions with 2.13.01.

Reply:
Also this change seems inappropriate for a commit titled "Merge freemarker property into uma".

Reply:
See also #12232.

Reply:
I'll move that change to a separate pull request.

Reply:
Done.

Reply:
> I don't know where the requirement for 2.13.03 came from; I was able to build all windows versions with 2.13.01.

I don't know either, but just because it builds doesn't mean it works, but I'll take it up in any new PR for this change.